packages/cf-builder-table/test/tableActions.js

```diff
@@ -1,8 +1,12 @@
-import { createMockStore } from 'cf-test-store';
-import cf from 'cf-pony';
-import { tableActions, TableActionTypes } from 'cf-builder-table';
+import { createMockStore } from '../../cf-test-store/src/index';
+import cf from '../../cf-pony/src/index';
+import {
+  tableActions,
+  TableActionTypes
+} from '../../cf-builder-table/src/index';
 
-test('.flashRow() should dispatch actions for flashing the row', done => {
+// this test is flaky, in 1/4 cases it just time outs
+test.skip('.flashRow() should dispatch actions for flashing the row', done => {
```

Comment:
I run `npm test -- --testPathPattern cf-builder-table` 10 times without flaky results. Let's hold on with skiping it